src/components/edit-asset-address-form/editable-address/editable-address.tsx

```diff
@@ -130,6 +133,31 @@ export const EditableAddress = ({
         setErrorHeading(locale.errors.unableToPatchAsset);
         setErrorDescription(locale.errors.tryAgainOrContactSupport);
       });
+
```

Comment:
So we're editing the tenure as well once an asset is edited, correct?

I think this block (137-159) should be in a new `.then` after the first one (line 116), before the catch (line 131), as this operation is dependent on the successful outcome of `patchAsset`.

I'd add another `.then` on line 130 and add this block within it so it should result in sometihing like this:

- Patch address API request
- If API request is successful, update current address state and show success message
- If API request is successful, send API request to update tenure
- If API request unsuccessful, show error heading